src/main/resources/overview.fxml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+<?import javafx.scene.shape.*?>
+<?import javafx.scene.text.*?>
+
+<fx:root alignment="TOP_CENTER" prefHeight="600.0" prefWidth="500.0" style="-fx-border-style: solid inside; -fx-border-color: grey; -fx-border-radius: 0; -fx-border-width: 10; -fx-background-color: white;" type="VBox" xmlns="http://javafx.com/javafx/10.0.2-internal" xmlns:fx="http://javafx.com/fxml/1" fx:controller="at.tugraz.sw20asd.lang.ui.Controller">
+
+    <AnchorPane prefHeight="602.0" prefWidth="800.0" AnchorPane.bottomAnchor="0" AnchorPane.leftAnchor="0" AnchorPane.rightAnchor="0" AnchorPane.topAnchor="0">
+    <children>
+
+    <Label fx:id="title" alignment="CENTER" layoutX="50.0" layoutY="73.0" maxWidth="-Infinity" prefHeight="42.0" prefWidth="350" text="OVERVIEW" textAlignment="CENTER">
+        <font>
+            <Font name="Abyssinica SIL" size="40.0" />
+        </font></Label>
+
+
+    <Button fx:id="add_btn" layoutX="50.0" layoutY="40.0" text="Add" GridPane.rowIndex="8">
+        <font>
+            <Font name="Abyssinica SIL" size="12.0" />
+        </font></Button>
+
+    <Button fx:id="vocabtrainer_btn"  layoutX="100.0" layoutY="450.0" text="Vocabulary Trainer" GridPane.rowIndex="8" >
+        <font>
+            <Font name="Abyssinica SIL" size="25.0" />
+        </font></Button>
+
+    <Button fx:id="edit_btn" layoutX="360.0" layoutY="40.0" text="Edit">
+        <font>
+            <Font name="Abyssinica SIL" size="12.0" />
+        </font></Button>
+        <Line endX="171.1016387939453" endY="-0.8284531831741333" layoutX="232.0" layoutY="128.0" startX="-180.19125366210938" startY="-0.8284209966659546" strokeMiterLimit="20.0" />
+
+    <TableView fx:id="table" layoutX="50.0" layoutY="140.0" prefWidth="350.0" prefHeight="250.0">
+        <TableCell>
+            <font>
+            <Font name="Abyssinica SIL" size="12.0" />
+            </font></TableCell>
+        <columnResizePolicy><TableView fx:constant="CONSTRAINED_RESIZE_POLICY"/></columnResizePolicy>
+        <columns>
+            <TableColumn prefWidth="75.0" text="German" />
```

Comment:
As a matter of extensibility, table headings should not be hard-coded like that. I think in a currently worked-on feature this will be fixed.